src/adagrams.js

```diff
@@ -1,15 +1,94 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  var letters = {
```

Comment:
I'm sure you know this now but we want to avoid using `var`, I think here we could use `const` instead.